src/Components/ProvisioningWizard/index.js

```diff
@@ -21,7 +22,7 @@ const ProvisioningWizard = ({ isOpen, onClose, image, ...props }) => {
   const [isConfirming, setConfirming] = React.useState(false);
   const [successfulLaunch, setLaunchSuccess] = React.useState();
 
-  const { isLoading, error: sourcesError, sources: availableSources } = sourcesForImage(image, { refetch: stepIdReached <= 2 });
+  const { isLoading, error: sourcesError, sources: availableSources } = useSourcesForImage(image, { refetch: stepIdReached <= 2 });
```

Comment:
why do we refetch every 10 seconds?

Reply:
We want to refetch to cover the case where user had filled created a Source in a tab we open after clicking the "Create Source" button.